.github/workflows/test-distros.yml

```
@@ -75,8 +75,11 @@ jobs:
       - name: Display Python version
         run: python3 -c "import sys; print(sys.version); print(sys.platform)"
 
-      - name: Install tox
-        run: pip3 install tox --break-system-packages
+      - name: Install uv
+        run: pip3 install uv --break-system-packages
```

GH runners are ephemeral so it does really matter but... just for better "role-modelling" maybe?
```suggestion
        run: pipx install uv
```
Only if pipx is available by default, otherwise forget it.